lib/httpInterface.js

```diff
@@ -342,6 +360,24 @@ function handleGetList(request, reply) {
       reply.getHttpError(INTERNAL_SERVER_ERROR_STATUS_CODE, error.message || 'something went wrong')
     })
     .pipe(this.castResultsAsStream())
+    .pipe(through2.obj((chunk, _, callback) => {
+      const lookupPaths = JSONPath({ json: chunk, resultType: 'pointer', path: '$..[?(@ !== null && @ !== undefined && @.label && @.value)]' })
+        .map(path => path
+          .split('/')
+          .filter(str => str.length > 0)
+          .join('.')
+        )
+      let castedChunk = {
+        ...chunk,
+        _id: chunk._id.toString(),
+      }
+      lookupPaths.forEach(path => {
+        castedChunk = lset(castedChunk, path, { ...lget(castedChunk, path), value: castedChunk[path].value.toString() })
```

Comment:
This should work the same:
```suggestion
        castedChunk = lset(castedChunk, `${path}.value`, castedChunk[path].value.toString())
```